src/components/screens/Makai/modals/LeverageModal/Body.tsx

```diff
@@ -353,7 +362,49 @@ export const LeveragerModalBody: FC<LeveragerModalBodyProps> = ({
                   <WarningDiv>
                     <p>Below liquidation threshold.</p>
                   </WarningDiv>
-                )}
+                )
+              }
+              <DetailsDiv>
+                <DetailDiv>
+                  <span>
+                    <TooltipMessage message="The ratio between LDOT and DOT at which your position would fall below the liquidation threshold." />
+                    Liquidation price:
+                  </span>
+                  <span>
+                    {
+                      (Number(leverage
```

Comment:
Could you make this fomula as a function?
Or this value can be calculated with useMemo